lib/slack_poster.rb

```diff
@@ -8,6 +8,8 @@ def initialize(webhook_url, team_channel, mood)
     @webhook_url = webhook_url
     @team_channel = team_channel
     @mood = mood
+    @today = Date.today
+    @postable_day = !today.saturday? && !today.sunday?
```

Comment:
I'd probably prefer positive predicates (such aliteration!) and configure _work_ days, but yes!